Dockerfile

```
@@ -138,5 +138,10 @@ RUN --mount=type=cache,target=/root/.cache/pip \
 
 ENV VLLM_USAGE_SOURCE production-docker-image
 
+# https://docs.docker.com/reference/dockerfile/#healthcheck
+# https://hub.docker.com/r/willfarrell/autoheal/
+HEALTHCHECK --interval=5m --timeout=10s \
+ CMD curl -f http://localhost/health || exit 1
```

nit but 4 spaces indent is standard for this file
```suggestion
    CMD curl -f http://localhost/health || exit 1
```